Workouts/SupportingViews/FeaturedWorkout.swift

```diff
@@ -25,13 +25,12 @@ struct FeaturedWorkout: View {
                         .foregroundColor(.gray)
                 }
             }
+            .padding(.top)
             MapView(workout: workout)
                 .frame(height: 200)
-                .cornerRadius(12.0)
-            Divider()
-                .padding(.top)
+                .padding(.horizontal, -15)
+                .padding(.bottom, -6)
```

Comment:
@emilyCheroske Had to do some weird negative padding stuff to account for the space the disclosure indicator is no longer taking up.